tests/otagent/src/test/java/org/hawkular/apm/tests/otagent/client/http/JBossRESTEasyClientITest.java

```diff
@@ -0,0 +1,223 @@
+        // affecting others
+        OpenTracingManager.reset();
+    }
+
+    @Test
+    public void testJaxRSClientGET() {
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").get();
+
+        processResponse(response, true, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientGETWithQS() {
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL_WITH_QS);
+        Response response = target.request().header("test-header", "test-value").get();
+
+        processResponse(response, true, false, true);
+    }
+
+    @Test
+    public void testJaxRSClientGETWithData() {
+        setProcessContent(true);
+
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").get();
+
+        processResponse(response, true, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientPOST() {
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").post(Entity.<String> text(SAY_HELLO));
+
+        processResponse(response, false, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientPOSTWithData() {
+        setProcessContent(true);
+
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").post(Entity.<String> text(SAY_HELLO));
+
+        processResponse(response, false, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientPUT() {
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").put(Entity.<String> text(SAY_HELLO));
+
+        processResponse(response, false, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientPUTWithData() {
+        setProcessContent(true);
+
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-header", "test-value").put(Entity.<String> text(SAY_HELLO));
+
+        processResponse(response, false, false, false);
+    }
+
+    @Test
+    public void testJaxRSClientGETWithFault() {
+        Client client = ClientBuilder.newClient();
+        WebTarget target = client.target(SAY_HELLO_URL);
+        Response response = target.request().header("test-fault", "true").get();
+
+        processResponse(response, true, true, false);
+    }
+
+    protected void processResponse(Response response, boolean get, boolean fault, boolean qs) {
```

Comment:
Removed 'get' parameter. `qs` is used to indicate that a query string has been supplied and should be checked.